lib/IRGen/GenEnum.cpp

```diff
@@ -265,9 +265,10 @@ static void emitResilientTagIndex(IRGenModule &IGM,
                                   const EnumImplStrategy *strategy,
                                   EnumElementDecl *Case) {
   auto resilientIdx = strategy->getTagIndex(Case);
-  auto global = IGM.getAddrOfEnumCase(Case, ForDefinition);
-  cast<llvm::GlobalVariable>(global.getAddress())
-      ->setInitializer(llvm::ConstantInt::get(IGM.Int32Ty, resilientIdx));
+  auto *global = cast<llvm::GlobalVariable>(
+    IGM.getAddrOfEnumCase(Case, ForDefinition).getAddress());
```

Comment:
This isn't the only use of `getAddrOfEnumCase`. Maybe *that's* what should be setting this as constant?

Reply:
This is the use where they're defined -- am I correct in thinking that constant or not doesn't matter for not-for-definition globals?

For field offsets for example, it might be constant in the defining module, and outside the module we don't really know if it was constant or not there, because of resilience.

Reply:
I suspect it's still used for CSE-like optimizations in LLVM.

Reply:
Fixed.